scripts/utils/setup-roles.ts

```diff
@@ -147,21 +163,21 @@ export async function main(
 
     // renounce ownership from deployer
     const renounceAdmin = await loanCore.renounceRole(ADMIN_ROLE, await deployer.address, {
-        gasLimit,
+        gasLimit
```

Comment:
Also for these `renounceRole` calls - no need to use `await deployer.address`. There is nothing to await since we are just reading a property?